Upstream: No
Reason: allow more liberal versions to be used

--- solidpython-1.0.2.orig/PKG-INFO
+++ solidpython-1.0.2/PKG-INFO
@@ -21,8 +21,8 @@
 Classifier: Topic :: Scientific/Engineering :: Mathematics
 Classifier: Topic :: Software Development :: Libraries :: Python Modules
 Requires-Dist: PrettyTable (==0.7.2)
-Requires-Dist: euclid3 (>=0.1.0,<0.2.0)
-Requires-Dist: pypng (>=0.0.19,<0.0.20)
-Requires-Dist: regex (>=2019.4,<2020.0)
+Requires-Dist: euclid3 (>=0.1.0)
+Requires-Dist: pypng (>=0.0.19)
+Requires-Dist: regex (>=2019.4)
 Project-URL: Documentation, https://solidpython.readthedocs.io/en/latest/
 Project-URL: Repository, https://github.com/SolidCode/SolidPython
--- solidpython-1.0.2.orig/setup.py
+++ solidpython-1.0.2/setup.py
@@ -13,9 +13,9 @@
 
 install_requires = \
 ['PrettyTable==0.7.2',
- 'euclid3>=0.1.0,<0.2.0',
- 'pypng>=0.0.19,<0.0.20',
- 'regex>=2019.4,<2020.0']
+ 'euclid3>=0.1.0',
+ 'pypng>=0.0.19',
+ 'regex>=2019.4']
 
 setup_kwargs = {
     'name': 'solidpython',
